packages/ref-metrics-ilamb/src/cmip_ref_metrics_ilamb/example.py

```diff
@@ -71,8 +71,8 @@ class GlobalMeanTimeseries(Metric):
     Calculate the global mean timeseries for a dataset
     """
 
-    name = "Global Mean Timeseries"
-    slug = "global-mean-timeseries"
+    name = "ILAMB Global Mean Timeseries"
```

Comment:
I don't think that the slug had to be globally unique. The intention was that it is unique for a provider. The constraint is that the combination of the provider slug + the metric slug was unique

Reply:
I had a bunch (~10) failed tests due to some errors which reported 'unique slug' problems that were fixed when I made the change. But I will check this again.